test/integration/fuzz/lockup/withdraw.t.sol

```diff
@@ -49,6 +48,39 @@ abstract contract Withdraw_Integration_Fuzz_Test is Integration_Test, Withdraw_I
         assertEq(actualWithdrawnAmount, expectedWithdrawnAmount, "withdrawnAmount");
     }
 
+    /// @dev Given enough fuzz runs, all of the following scenarios will be fuzzed:
+    ///
+    /// - Multiple caller addresses.
+    function testFuzz_Withdraw_UnknownCaller(address caller)
+        external
+        whenNotDelegateCalled
+        givenNotNull
+        whenToNonZeroAddress
+        whenWithdrawAmountNotZero
+        whenWithdrawAmountNotGreaterThanWithdrawableAmount
+    {
+        vm.assume(caller != users.sender && caller != users.recipient);
```

Comment:
Why are we excluding the sender and the recipient?

Reply:
well, the test name is `testFuzz_Withdraw_UnknownCaller`

notice _UnknownCaller_

Reply:
LOL. I had a brain glitch!